graphql_client/README.md

```diff
@@ -0,0 +1,11 @@
+# graphql_client
+
+[![Build Status](https://travis-ci.org/graphql-rust/graphql-client.svg?branch=master)](https://travis-ci.org/graphql-rust/graphql-client)
+[![docs](https://docs.rs/graphql_client/badge.svg)](https://docs.rs/graphql_client/latest/graphql_client/)
+[![crates.io](https://img.shields.io/crates/v/graphql_client.svg)](https://crates.io/crates/graphql_client)
+[![Join the chat](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/juniper-graphql/graphql-client)
+
+A typed GraphQL client library for Rust.
+
+## Refer to the root README
+https://raw.githubusercontent.com/graphql-rust/graphql-client/master/README.md
```

Comment:
Great, thanks for the PR! Can we make this line a link? Maybe something like "The library is documented in the [README at the root of the project](https://github.com/graphql-rust/graphql-client)"

Reply:
@luckycoder1030 
Could you fix it?